modules/nw-ovn-kuberentes-change-join-subnet.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configure-ovn-kubernetes-subnets.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-change-join-subnet_{context}"]
+= Configuring the OVN-Kubernetes join subnet
+
+You can change the join subnet used by OVN-Kubernetes.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Ensure that the cluster uses the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. To change the OVN-Kubernetes join subnet, enter the following command:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io cluster --type='merge' \
+  -p='{"spec":{"defaultNetwork":{"ovnKubernetesConfig":
+    {"ipv4":{"internalJoinSubnet": "<join_subnet>",
+    "ipv6":{"internalJoinSubnet": "<join_subnet>"}}}}}}'
+----
++
+--
+where:
+
+`<join_subnet>`:: Specifies an IP address subnet for internal use by OVN-Kubernetes. The subnet must be larger than the number of nodes in the cluster and it must be large enough to accommodate one IP address per node in the cluster. This subnet cannot overlap with any other subnets used by {product-title} or on the host itself. The default value for IPv4 is `100.64.0.0/16` and for IPv6 it is `fd98::/64`.
+--
++
+.Example output
+[source,text]
+----
+network.operator.openshift.io/cluster patched
+----
+
+. To confirm that the configuration is active, enter the following command. It can take up to 30 minutes for this change to take effect.
++
+[source,terminal]
+----
+$ oc get network.operator.openshift.io \
+  -o jsonpath="{.items[0].spec.defaultNetwork}"
```

Comment:
Recommend adding an example output block, so that people know what to look for to know if it has indeed taken effect.